### DISCUSSION TITLE
Encountring Error while developing my own tailwind snippets extension

### DISCUSSION THREAD

**Discussion by SidakVats** [microsoft/vscode-discussions#1235] [Extension Development QnA] [answered]
my repo link :  https://github.com/SidakVats/tailwind-snippets

I have made my own vs code tailwind snippets extension

- In the terminal, I executed the following commands:

 1.) vsce login <publishername>
 2.) vsce package (which successfully created a .vsix file)
 3.) vsce publish
 
 but then this appears on terminal :
  WARNING   LICENSE.md, LICENSE.txt or LICENSE not found
  Do you want to continue? [y/N] y
 INFO   Publishing 'SidakVats.tailwind-react-snippetss v0.0.1'...
 ERROR    Your extension has suspicious content. Please fix your extension metadata, or contact support if you need assistance.

**Comment by isidorn** [comment, marked as answer]
This is a valid warning. Please include a LICENSE.txt
You can also start from some samples if that will make this easier https://github.com/microsoft/vscode-extension-samples
